pkg/virt-api/rest/subresource.go

```diff
@@ -72,6 +80,25 @@ const (
 type validation func(*v1.VirtualMachineInstance) error
 type URLResolver func(*v1.VirtualMachineInstance, kubecli.VirtHandlerConn) (string, error)
 
+type Informers struct {
+	PodInformer cache.SharedIndexInformer
+}
+
+func NewInformer() *Informers {
+	kubeClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
+	}
+	namespace, err := clientutil.GetNamespace()
+	if err != nil {
+		glog.Fatalf("Error searching for namespace: %v", err)
+	}
+	kubeInformerFactory := controller.NewKubeInformerFactory(kubeClient.RestClient(), kubeClient, namespace)
```

Comment:
The namespace here would limit the informer to only watch on the namespace of `virt-api`, not on all namespaces. The informerfactory above is already correctly configured. You don't need another one.